docs/content/guides/author-apps/custom/howto-extenders/index.md

```diff
@@ -0,0 +1,78 @@
+---
+type: docs
+title: "How-To: Model any resource with an extender"
+linkTitle: "Extenders"
+description: "Learn how to use extenders in an application"
+weight: 300
+---
+
+This guide will walk you though how to use extenders in an application to model any resource.
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+
+## Step 1: Register an extender Recipe
+
+In this guide you will leverage a [Recipe]({{< ref "/guides/recipes/overview" >}}) to deploy backing infrastructure for your resource. Begin by registering the 'extender-postgresql' Recipe in your environment:
+
+```bash
+rad recipe register postgresql --resource-type "Applications.Core/extenders" --template-kind bicep --template-path "ghcr.io/radius-project/recipes/local-dev/postgresql:latest"
+```
+
+## Step 2: Define an extender
+
+Open a new file named `app.bicep` and define an extender:
+
+{{< rad file="snippets/app.bicep" embed=true >}}
+
+## Step 3: Add a container
+
+Add a container to your `app.bicep` file, accessing the extender's properties and secrets:
+
+{{< rad file="snippets/app-container.bicep" embed=true marker="//CONTAINER" >}}
+
+## Step 4: Deploy the app
+
+Run `app.bicep` with [`rad run`]({{< ref rad_run >}}):
```

Comment:
I prefer to keep the actual command within the bash block to prevent users from copying-pasting incomplete commands:

```suggestion
Deploy and [run]({{< ref rad_run >}}) the app using the following command:
```